spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/web/embedded/TomcatWebServerFactoryCustomizerTests.java

```diff
@@ -92,6 +93,13 @@ public void customBackgroundProcessorDelay() {
 	@Test
 	public void customDisableMaxHttpPostSize() {
 		bind("server.tomcat.max-http-post-size=-1");
+		bind("server.tomcat.max-http-form-post-size=-1");
```

Comment:
Changing an existing test feels a little `strange`. Curious if I am doing anything wrong. These tests should pass regardless of the new additions. No? 🤔

Reply:
We should probably have two tests to make sure that the old property still works. As you've added `customDisableMaxHttpFormPostSize` below, this test should be left unchanged.